components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -249,11 +250,17 @@ - (void)layoutSubviews {
   titleCenter.x = CGRectGetMidX(contentBounds);
   titleBounds.size = titleSize;
 
-  // Horizontally align the badge.
-  CGSize badgeSize = [_badgeLabel sizeThatFits:contentBounds.size];
-  badgeSize.width = MIN(kBadgeMaxWidth, badgeSize.width);
+  // Size badge
+  NSString *longestAllowableBadgeString = [_badgeLabel.text substringToIndex:kBadgeMaxTextLength];
```

Comment:
It looks like this counts non-printable characters, but I suspect it should not. Is there any reason to limit to 3 when the material.io examples for Bottom Navigation show 4 characters?

https://material.io/design/components/bottom-navigation.html#behavior
<img width="616" alt="screen shot 2019-03-06 at 1 16 15 pm" src="https://user-images.githubusercontent.com/1753199/53903578-09d42d80-4012-11e9-9086-a4063aee1b8b.png">

Reply:
The documentation for the constant `kBadgeMaxWidth` that I'm deleting implied that it should be three characters, but this picture makes it look like it's four. I can change it to four.